hosted-ce/etc/osg/image-config.d/30-remote-site-setup.sh

```diff
@@ -156,8 +153,11 @@ if [[ -n $BOSCO_GIT_ENDPOINT && -n $BOSCO_DIRECTORY ]]; then
 fi
 unset GIT_SSH_COMMAND
 
+users=$(get_mapped_users)
+[[ -n $users ]] || errexit "Did not find any HTCondor-CE SCITOKENS user mappings"
+
 # Setup sudoers.d file
-setup_sudo_users
+setup_sudo_users $users
```

Comment:
```suggestion
setup_sudo_users "$users"
```

Protect yourself before you wreck yourself. Shell expansion could result in multiple args if `$users` is whitespace separated instead of a single, whitespace-delimited string arg

Reply:
Good catch, and thanks for bearing with me... my bash is not the best